docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,239 @@
+#####################################################
+Compute circuit output probabilities with ``Sampler``
+#####################################################
+
+This guide shows how to get the probability distribution of a quantum circuit with the :class:`~qiskit.primitives.Sampler` primitive.
+
+.. note::
+
+    While this guide only uses Qiskit Terra's implementation of the ``Sampler`` primitive, there are other
+    implementations of this primitive like Qiskit Terra's :class:`~qiskit.primitives.BackendSampler`, Qiskit Aer's :class:`~qiskit_aer.primitives.Sampler`
+    and Qiskit Runtime's :class:`~qiskit_ibm_runtime.Sampler`.
+
+Initialize quantum circuits
+===========================
+
+The first step is to create the :class:`~qiskit.circuit.QuantumCircuit` from which you want to obtain the probability distribution.
+
+.. plot::
+    :include-source:
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.measure_all()
+    qc.draw("mpl")
+
+.. testsetup::
+
+    from qiskit import QuantumCircuit
+
+    qc = QuantumCircuit(2)
+    qc.h(0)
+    qc.cx(0,1)
+    qc.measure_all()
+
+.. note::
+
+    The :class:`~qiskit.circuit.QuantumCircuit` you pass to :class:`~qiskit.primitives.Sampler` has to include measurements.
+
+Initialize the ``Sampler``
+==========================
+
+Then, you need to create a :class:`~qiskit.primitives.Sampler` object.
```

Comment:
```suggestion
Then, you need to create a :class:`~qiskit.primitives.Sampler` instance.
```

Maybe you can explain here `sampler = BackendSampler(backend)` to wrap a backend that does not support it natively. I assume that if a provider implements it natively, you can `from ... import Sampler`.